### ISSUE TITLE
Installing - editing paths to circos-tools

### ISSUE THREAD

**Issue by jrjhealey** [infspiredBAT/Circoletto#2]
Hi guys,

I was wondering if you have any information on how to access the `circos-tools` that the script needs. I've not been able to find any information about this online (though installing circos and BioPerl is relatively trivial), and there is nothing in the README.

Cheers

**Comment by nikosdarzentas** [comment]
Hello,

There's `$path2circos` and `$path2circostools` ca line 120 in the main script.
There's also a single line instruction when you run the main script without any arguments, hinting at it:
"_check / edit (in the code) the two paths to Circos and Circos tools_"
Don't get me wrong, I know that our documentation is not really, well, there... :-)
I've made a note to improve this, even exposing those two variables as input arguments.

Many thanks for getting in touch and for trying Circoletto - hope you do find it useful.

Best wishes.

**Comment by jrjhealey** [comment]
Hi Nikos,

Yep, its not the path editing thats the problem, its that I don't actually know what `circos-tools` is, or where to get it?

Is it a completely separate install or is it within circos some where?

**Comment by jrjhealey** [comment]
Hi Nikos, I have subsequently found it.

For future reference, `circos-tools` in `apt-get` is currently non-functional (requests time out). I had to download it directly from Circos (http://circos.ca/software/download/tools), but for some reason it's not easy to find in google so maybe this will be useful for others.

One last thing though, the option to override the max number of seqs isnt a documented commandline option in the help output (though it does say that circos' housekeeping.conf will need editing). Is this deliberate?

**Comment by nikosdarzentas** [comment]
Hey Joe (Jimmy Hendrix ref),

Ah OK, I understand now - I'm on vacation and therefore not focused enough.
Anyway, I'm glad you figured it out.

And I don't believe that's deliberate, it's just how the script evolved and now is, or was:
I updated the link to `circos` and `circos-tools` to your URL that contains both,
I exposed the max_sequences and max_ribbons arguments to command line
 - I'll soon update the main script here on GitHub.

Many thanks for your time and feedback.
